### DISCUSSION TITLE
(synced-lyrics) Too small text in any "Line Effect" mode, excluding "Fancy"

### DISCUSSION THREAD

**Discussion by Dzheremi2** [pear-devs/pear-desktop#2778] [Q&A] [answered]
After update 3.7.0, in every line effect except fancy the text became too small. Is this expected behavior?
![Снимок экрана от 2024-12-26 16-39-57](https://github.com/user-attachments/assets/b2c5e3e6-2152-495f-8df1-5317b6c81225)
@ArjixWasTaken

**Comment by JellyBrick** [comment]
@kimjammer

**Comment by kimjammer** [comment]
That's the size I remember it being. Modes other than fancy should have their font size match the official music.youtube.com interface, could you compare it to that?

**Reply by Dzheremi2** [reply]
It matches the official. But this app isn't official. Why did you decide that font size should match the official one? Previous font size was good for reading, the new one is too small, and it needed to take a closer look to read it. Anyway `Fancy` mode font was bigger than previous one, so what's the reasong to change it?

**Reply by kimjammer** [reply]
I'm not sure why the font size changed for you, it didn't change for me. As far as I could tell, the font size was never meant to be bigger on previous versions, as it just used the default font size that comes with youtube music. Did you have a custom theme or css that changed the font sizes?

**Reply by Dzheremi2** [reply]
I didn't have any CSS theme installed. It just changed after 3.7.0 update. This is how it looked like in 3.6.2
![Снимок экрана от 2024-12-26 23-35-09](https://github.com/user-attachments/assets/cc4b34b8-fd33-4910-8ea8-51b8ff09a2fe)

**Reply by Dzheremi2** [reply]
Also, if you'll see the screenshot made by ArjixWasTaken (the author of the plugin) font WAS meant to be as in 3.6.2
![](https://private-user-images.githubusercontent.com/53124886/379873595-4a9323b3-f6b6-4beb-914f-2eb240c086ef.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUyNDY5ODgsIm5iZiI6MTczNTI0NjY4OCwicGF0aCI6Ii81MzEyNDg4Ni8zNzk4NzM1OTUtNGE5MzIzYjMtZjZiNi00YmViLTkxNGYtMmViMjQwYzA4NmVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI2VDIwNTgwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE5OWRmOTNkMTE1YTBjNzhkOGIxYTgwYWUwMjY0MTE1ZGIyYjU5ZDM2NDNhMDI0ODRiODI2NGNmMGNhYTExZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.u8_xAvMCQ852eeHsWu6TtuyvdThY2hdnCMlts4mx9iY)

**Reply by kimjammer** [reply]
Interesting, I'll take a look once I'm back

**Reply by Dzheremi2** [reply]
Thanks for your understanding

**Reply by ArjixWasTaken** [reply]
Sounds like our two PRs did not align well after the merge, I will be free to dedicate some hours tomorrow on this, but I can't promise much due to uni deadlines.

PS: It may seem like I am always busy with my uni, but it is the opposite of that, I fail to follow a schedule so every 3 months I have to face the consequences and study a lot + do a lot of homework :pensive:

**Reply by kimjammer** [reply, marked as answer]
I found the issue and made the fix. The font-size used to scale between 1.4rem (official size) and 3rem based on the width of the app window! I didn't notice it because the font size literally didn't change for me 😅, while they changed for you.

**Reply by Dzheremi2** [reply]
Thanks for your research and fast fix!

**Reply by kimjammer** [reply]
Tracking on #2788

**Reply by Dzheremi2** [reply]
This is kinda funny that I've never minimized the window and you (probably) never maximized it to see font changes.🙃
